addons/csw/CfgVehicles.hpp

```diff
@@ -29,21 +33,27 @@ class CfgVehicles {
         EGVAR(dragging,carryPosition)[] = {0, 2, 0};
 
         class ACE_Actions {
-            class ACE_MainActions {
-                displayName = CSTRING(genericTripod_displayName);
+            class ADDON {
```

Comment:
This gave me problems with overlapping interactions,
only saw main actions and couldn't deploy


<img width="665" height="424" alt="Image" src="https://github.com/user-attachments/assets/318a5259-11c4-4d59-bb61-fb5fe83bc440" />

could do
```
class ACE_MainActions {
                selection = "";
                distance = 2.5;
                condition = "true";
            class ADDON {
                displayName = CSTRING(AddonName_Short);
                condition = QUOTE(call FUNC(isModEnabled));
...
```
to put it under main
or just add `call FUNC(isModEnabled)` to the conditions and keep the rest the same as before

Reply:
ace actions fuck me up